ts/client/src/accounts/mangoAccount.ts

```diff
@@ -378,22 +378,20 @@ export class MangoAccount {
 
   /**
    * Sum of all positive assets.
-   * TODO: healthType = HealthType.maint breaks UI assumption
    * @returns assets, in native quote
    */
-  public getAssetsValue(group: Group, healthType = HealthType.maint): I80F48 {
+  public getAssetsValue(group: Group): I80F48 {
     const hc = HealthCache.fromMangoAccount(group, this);
-    return hc.assetsAndLiabs(healthType, true).assets;
+    return hc.healthAssetsAndLiabs(undefined, true).assets;
   }
 
   /**
    * Sum of all negative assets.
-   * TODO: healthType = HealthType.maint breaks UI assumption
    * @returns liabs, in native quote
    */
-  public getLiabsValue(group: Group, healthType = HealthType.maint): I80F48 {
+  public getLiabsValue(group: Group): I80F48 {
     const hc = HealthCache.fromMangoAccount(group, this);
-    return hc.assetsAndLiabs(healthType, false).liabs;
+    return hc.healthAssetsAndLiabs(undefined, false).liabs;
```

Comment:
I think getAssetsValue and getLiabsValue need to use a consistent version of healthAssetsAndLiabs(), otherwise comparing the two doesn't make sense (and the ui will show them next to each other)

Reply:
Ah, if all weights are set to 1, the both versions are the same!

Reply:
I sitll made them consistent